openapscontrib/predict/predict.py

```diff
@@ -163,7 +174,7 @@ def future_glucose(
         last_history_datetime = parse(last_history_event['end_at'])
         simulation_end = max(simulation_end, last_history_datetime)
 
-    simulation_end += datetime.timedelta(hours=insulin_action_curve)
+    simulation_end += datetime.timedelta(minutes=(insulin_action_curve * 60 + sensor_delay))
```

Comment:
You can actually do `timedelta(hours=insulin_action_curve, minutes=sensor_delay)` here